docker/unit_tests.sh

```diff
@@ -32,8 +32,6 @@ localize_apps: true
 dashboard_enable_pegasus: true
 dashboard_workers: 5
 skip_seed_all: true
-optimize_webpack_assets: false
-optimize_rails_assets: false
```

Comment:
LGTM after adding a comment here explaining how we might be able to squeeze a few minutes out of the build time here.